tools/bin/ariaDownload.py

```diff
@@ -62,28 +65,28 @@ def __init__(self, inps):
         self.inps.output = self.inps.output.title()
         self.inps.wd     = op.abspath(self.inps.wd)
         self.url_base    = 'https://api.daac.asf.alaska.edu/services/search/param?'
+        if self.inps.v: log.setLevel('DEBUG')
+        return
 
     def __call__(self):
         url              = self.form_url()
         dict_prod, urls  = self.parse_json(url)
         script           = requests.post(f'{self.url_base}&output={self.inps.output}', data=dict_prod).text
 
-
         if self.inps.output == 'Count':
-            print (f'\nFound -- {len(urls)} -- products')
-
+            log.info (f'\nFound -- {len(urls)} -- products')
```

Comment:
Remove space, let logger do formatting
```suggestion
            log.info('\nFound -- %d -- products', len(urls))
```

Reply:
(same for the rest in this file)